data-visualization/json/index02.js

```diff
@@ -0,0 +1,21 @@
+let myTeam = {};
```

Comment:
```suggestion
const myTeam = {};
```
`const` key word change

From here on I wont suggest the `const` change -- make those changes locally and push them up (or if we decide to use `let` then ignore 😁)